HopsanGUI/Dialogs/OptimizationScriptWizard.cpp

```diff
@@ -169,6 +169,10 @@ OptimizationScriptWizard::OptimizationScriptWizard(SystemObject* pSystem, QWidge
     mpMPLineEdit = new QLineEdit("0.1", this);
     mpMPLineEdit->setValidator(new QDoubleValidator());
 
+    mpElitesLabel = new QLabel("Number of elites: ");
+    mpElitesLineEdit = new QLineEdit("4", this);
+    mpElitesLineEdit->setValidator(new QDoubleValidator());
```

Comment:
DoubleValidator used here, but the number is not a double, should be int ?

Reply:
Fixed!